doc/src/_static/custom.css

```diff
@@ -143,3 +143,45 @@ This can be removed once https://github.com/executablebooks/mdit-py-plugins/issu
 .task-list-item-checkbox {
     margin: 0 0.2em 0.25em -1.4em;
 }
+
+/* Some styles for the TryExamples directive buttons enabled by jupyterlite-sphinx */
+/* The accessibility results are as follows:
+/* https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=22632F, and
+/* https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=348A47
+/* where we make them pass on the "Large Text" scale for AA and AAA, since
+/* the font is weighted is considered enough to be large text. */
```

Comment:
Are these necessary to make it match the theme? It seems like some of these should be part of jupyterlite-sphinx.

Reply:
Yes, I did suggest doing so previously, though the general opinion was not to do so, as it would require extra maintenance with corresponding style updates in themes upstream, such as PST, or for the Furo theme here. Also, SymPy has put down some customisations on top of Furo as well, so our goal is to provide what works for all – which is intrinsically to provide nothing.

This was also discussed in slightly more detail here: https://github.com/numpy/numpy/pull/26745#issuecomment-2283488118